packages/perseus/src/widgets/radio/multiple-choice-option.new.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useRef} from "react";
+
+import Indicator from "./choice-indicator.new";
+import styles from "./choice.module.css";
+
+export interface IndicatorProps {
+    checked: boolean;
+    children: React.ReactNode | React.ReactNode[];
+    indicatorContent: string;
```

Comment:
I wonder if a comment would help here, to explain that the indicatorContent is the letter of the choice